GameServer.py

```diff
@@ -14,6 +14,7 @@ class GameServer(Server):
     def __init__(self, *args, **kwargs):
         Server.__init__(self, *args, **kwargs)
         self.players = WeakKeyDictionary()
+        self.bombs = 0
```

Comment:
Do we need information on server how many bombs are there?

Reply:
No we don't, already taken care of.